pkg/virtctl/pause/pause.go

```diff
@@ -122,7 +121,13 @@ func (vc *VirtCommand) Run(cmd *cobra.Command, args []string) error {
 			}
 			err = virtClient.VirtualMachineInstance(namespace).Pause(vmiName)
 			if err != nil {
-				return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
+				if vm.Spec.Running != nil && !*vm.Spec.Running {
```

Comment:
I would start with a check if `err` is a NotFound error. Only then it makes sense to check if the VM is running IMHO.
For the running check then you also need to check the `vm.Spec.RunStrategy`.

Reply:
Makes sense and is more readable IMO. Thanks